### PR TITLE
Allow running as a cluster without Redis

### PR DESCRIPTION
**Pull request by BenLubar**
Redis is still used for pubsub if it is present.

Extracted from #5861.

### CODE REVIEW

**Comment by nodebb-misty** [comment]
This pull request was made against the `develop` branch, which while not strictly disallowed, is unorthodox. If this is intentional, please respond back with the reason for this pull request being made to the develop branch.

Thanks!

**Comment by barisusakli** [comment]
I think this changes the behavior so that a pubsub is always used even when it is not needed. When you only have 1 nodebb process we don't need to use any pubsub, can just emit like it was before.

**Comment by BenLubar** [comment]
@barisusakli that is correct. It should have the old behavior for non-clustered installations now.

File: src/pubsub.js [hunk 1]
```diff
@@ -27,7 +27,11 @@ function get() {
 
 	var pubsub;
 
-	if (nconf.get('redis')) {
+	if (nconf.get('isCluster') === 'true') {
```
Comment (barisusakli) [inline src/pubsub.js]: shouldn't it be `nconf.get('isCluster') === 'false'`

Comment (BenLubar) [reply src/pubsub.js]: Whoops! Good catch!

**Comment by BenLubar** [comment]
Rebased and fixed the merge conflict.

**Pull request merged**